api/external/events.html

```diff
@@ -0,0 +1,530 @@
+                                                        <li><strong>event_type</strong> (<em>string</em>) &#8211; The event type display name.</li>
+                                                        <li><strong>event_monetary_value</strong> (<em>number</em>) &#8211; A monetary value associated with the event.</li>
+                                                        <li><strong>mailing_id</strong> (<em>string</em>) &#8211; A mailing ID to associate with the event.</li>
+
+                                                    </ul>
+
+                                                </li>
+                                            </ul>
+                                        </td>
+                                    </tr>
+                                    <tr class="field-even field">
+                                        <th class="field-name">Returns:</th>
+                                        <td class="field-body">
+                                            <p class="first">Information for the new member events.</p>
+                                        </td>
+                                    </tr>
+                                    <tr class="field-even field">
+                                        <th class="field-name">Raises:</th>
+                                        <td class="field-body">
+                                            <tt class="docutils literal">
+                                                <span class="pre">Http400</span>
+                                            </tt>
+                                            if no events are provided.
+                                        </td>
+                                    </tr>
+                                    </tbody>
+                                </table>
+                                <p>
+                                <div class="sample-response hide-response">
+                                    <span style="font-weight:bold;">Sample Response</span>
+                                    <a href="#" class="resp-trigger">[<span class="tshow">show</span><span class="thide">hide</span>]</a>
+                                    <pre class="response">
+    <b>POST /100/events/members</b>
+    [
+        {
+            "account_id": 100,
+            "event_timestamp": "@D:2025-12-02T11:23:47",
+            "event_type_id": 300,
+            "event_monetary_value": 8.0,
+            "inserted_at": "@D:2025-12-02T11:23:47"
+            "mailing_id": null,
+            "member_id": 555,
+        },
+        {
+            "account_id": 100,
+            "event_timestamp": "@D:2025-12-02T11:23:47",
+            "event_type_id": 300,
+            "event_monetary_value": 2.0,
+            "inserted_at": "@D:2025-12-02T11:23:47"
+            "mailing_id": null,
+            "member_id": 444,
+        }
+    ]
```

Comment:
An adjustment to the sample request body to show different variations of the accepted request format

```suggestion
    [
        {
            "event_timestamp": "@D:2025-12-02T11:23:47",
            "event_type_id": 300,
            "event_monetary_value": null,
            "mailing_id": null,
            "member_id": 555,
        },
        {
            "event_timestamp": "@D:2025-12-02T11:23:47",
            "event_type": "Purchased Tickets",
            "event_monetary_value": 2.0,
            "mailing_id": 201,
            "email": "emma@myemma.com",
        }
    ]
```